Packs/SOCRadar/Integrations/FeedSOCRadarThreatFeed/README.md

```diff
@@ -0,0 +1,151 @@
+Retrieve indicators provided by collections via SOCRadar Threat Intelligence Feeds.
+This integration was integrated and tested with v21.11 of SOCRadar.
+
+## Configure SOCRadar Threat Feed on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for SOCRadarThreatFeed.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | API Key | The API Key to use for connection to SOCRadar ThreatFusion API. | True |
+    | insecure | Trust any certificate (not secure). | False |
+    | proxy | Whether to use XSOAR’s system proxy settings to connect to the API. | False |
+    | Feed Name | The feed name(s) to fetch. | True |
+    | Fetch indicators | Whether to fetch indicators. | False |
+    | Indicator Reputation | Indicators from this integration instance will be marked with this reputation. | False |
+    | Traffic Light Protocol Color | The Traffic Light Protocol \(TLP\) designation to apply to indicators fetched from the feed. | False |
+    | Tags | Supports CSV values. | False |
+    | Source Reliability | Reliability of the source providing the intelligence data. | True |
+    | Feed Fetch Interval | The feed fetch interval. | False |
+    | Bypass exclusion list | When selected, the exclusion list is ignored for indicators from this feed. This means that if an indicator from this feed is on the exclusion list, the indicator might still be added to the system. | False |
+
+4. Click **Test** to validate API key and connection to SOCRadar Threat Feeds/IOC API.
+
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+
+### How to obtain SOCRadar Threat Feeds/IOC API key?
+
+Every company has a unique API key in SOCRadar platform. This API key can be used to benefit from
+various API endpoints that SOCRadar provides. 
+
+You can obtain your company's API key from your company's settings page by navigating to the 
+`API Options` tab. Besides, API key can be regenerated by using `Actions` button under the API 
+keys panel.
+
+![api_key](../../doc_files/api_key.png) 
```

Comment:
The screenshot of the API key obtaining process is used to help user for finding the API key on SOCRadar platform. We used the same screenshot in our previously implemented integration, "SOCRadar Incidents", so we thought it can be used in this one as well. But since we provide the link to the documentation for obtaining the API key, we can simply remove this screenshot as you suggested. 

Should we remove it from the SOCRadar Incidents integration as well?

Reply:
It's Ok, leave the SOCRadar Incidents as is. It's only a recommendation as I mentioned.